### ISSUE TITLE
unexpected worker invocations while using experimental-edge

### ISSUE THREAD

**Issue by jmonster** [cloudflare/next-on-pages#29]
TL;DR is it intended that all requests, including all static assets, are each billable function invocations?

I might be misunderstanding how CloudFlare is billing, but allow me to share my unexpected behavior when opting one route into `experiemental-edge` rendering. **If nothing else it might be nice to clarify the documentation to address this if everything is behaving as intended.**

Here are my routes per `next build`:

```sh
Route (pages)                              Size     First Load JS
┌ ○ /                                      9.54 kB        92.1 kB
├   /_app                                  0 B            78.9 kB
├ ○ /404                                   182 B          79.1 kB
├ ℇ /meme                                  183 B          89.7 kB
└ ○ /meme-without-props                    237 B          89.7 kB
+ First Load JS shared by all              82.1 kB
  ├ chunks/framework-7751730b10fa0f74.js   45.5 kB
  ├ chunks/main-d63a912fa970a2f1.js        31 kB
  ├ chunks/pages/_app-d6d1f1cf1c8ebe7b.js  606 B
  ├ chunks/webpack-07929ad31bf03138.js     1.86 kB
  └ css/9eeb3fd1401c3d46.css               3.13 kB
```

**I expected that only requests to `/meme` would trigger a Function** while the remaining static assets would be served similarly to the static nextjs buildpack.

However, loading my homepage increments my worker usage by exactly 17 requests -- identical to the total number of requests (images/icons/fonts/js/css/etc) made in the browser, all to static assets.

Here are the docs I find confusing: https://developers.cloudflare.com/pages/platform/functions/pricing/#free-plan
> Requests to static assets that do not include Functions are free and unlimited, regardless of your plan.

**Comment by KieranHolroyd** [comment]
@eelislynne
`next is compiled down to a single file _workers.js which then has full power to handle all http requests sent to pages
which is also the answer to ^^ the git question, as to why static resources take up worker requests, because the _workers.js file also handles static files`

**Comment by arielvieira** [comment]
@jmonster You can create a `_routes.json` file inside your `public` next folder with the static assets/page, so Cloudflare Pages know which routes should call as a function or as a static asset.

https://developers.cloudflare.com/pages/platform/functions/routing/#creating-a-_routesjson-file

**Comment by jmonster** [comment]
Excellent, thank you so much!
I'm now only seeing 1 worker invocation on page load with the following `_routes.json`:

```json
{
  "version": 1,
  "include": ["/*"],
  "exclude": [
    "/favicon.ico",
    "/manifest.json",
    "/fonts/*",
    "/images/*",
    "/_next/static/*"
  ]
}
```

Ideally I would see 0 since `/` is a static page, but I assume it's the nature of the beast when mixing static and dynamic content together? NextJS doesn't appear to export an `index.html` file unless you call `next export` which then wouldn't support having some dynamic pages.

_Please correct me if I'm wrong!_

**Comment by GregBrimble** [comment]
Closed as duplicate of #5
